exporter/prometheusremotewriteexporter/documentation.md

```diff
@@ -14,6 +22,14 @@ Number of translation operations that failed to translate metrics from Otel to P
 | ---- | ----------- | ---------- | --------- |
 | 1 | Sum | Int | true |
 
+### otelcol_exporter_prometheusremotewrite_sent_batch
```

Comment:
Based on https://github.com/open-telemetry/semantic-conventions/blob/e70853397b73ca9cb85b1df45182b6a3e299c85e/docs/general/naming.md#naming-rules-for-counters-and-updowncounters, it seems like this should be `batches`, rather than `batch`

Reply:
Should I also re-add count which I had previously removed thinking it was redudant?

Reply:
This is a Counter, not an updowncounter, so the updowncounter guidance does not apply here

> Metric names SHOULD NOT be pluralized, unless the value being recorded represents discrete instances of a [countable quantity](https://wikipedia.org/wiki/Count_noun). Generally, the name SHOULD be pluralized only if the unit of the metric in question is a non-unit (like {fault} or {operation}).

This is a count of discrete instances of a countable quantity, so it should be plural.

Reply:
Done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37912/commits/dd3c4665b21c07787e25cd573336fd2ed1d8c627

Appreciate the feedback - can you please kick off the CI again?